int32[] joints      # list of D joints to control
float64[] times     # N timestamps (in seconds)
float32[] torques   # NxD torque matrix
